patches/components-policy-resources-templates-policies.yaml.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/components/policy/resources/templates/policies.yaml b/components/policy/resources/templates/policies.yaml
+index ecde8eaa2306e9fc46d4f2de636a1c3fbd1fd536..5081dcddac551a4bd3032d757c767202f8e45165 100644
+--- a/components/policy/resources/templates/policies.yaml
++++ b/components/policy/resources/templates/policies.yaml
+@@ -1293,6 +1293,14 @@ policies:
+   1292: AdHocCodeSigningForPWAsEnabled
+   1293: KioskWebAppOfflineEnabled
+   1294: GraduationEnablementStatus
++  1295: TorDisabled
```

Comment:
I'm thinking about patching this with one entry:
`BRAVE_GROUP_POLICIES`

And then using a Chromium src override to replace that value with the ones in the `src/brave/components/policy/resources/templates/policy_definitions/BraveSoftware/` folder

Reply:
actually- found a better place. We can intercept where policies.yaml gets loaded in `src/components/policy/resources/policy_templates.py` 😄 Stay tuned

Reply:
Resolving - solved with latest commit